sklearn/naive_bayes.py

```diff
@@ -526,9 +526,14 @@ def _check_alpha(self):
                 raise ValueError("alpha should be a scalar or a numpy array "
                                  "with shape [n_features]")
         if np.min(self.alpha) < _ALPHA_MIN:
-            warnings.warn('alpha too small will result in numeric errors, '
-                          'setting alpha = %.1e' % _ALPHA_MIN)
-            return np.maximum(self.alpha, _ALPHA_MIN)
+            if self.force_alpha:
+                warnings.warn('alpha too small will result in numeric errors, '
+                              'force_alpha was set to True, '
+                              'proceeding without changing alpha.')
```

Comment:
I think that we can report the value of `alpha` in the warning, making it easier users to inspect its value in the case of numeric errors. Also here is a small rewording proposals (not using f-string for the consistency):
```suggestion
                warnings.warn('alpha too small will result in numeric errors. '
                              'Proceeding with alpha = %.1e, as '
                              'force_alpha was set to True.' % alpha)
```

Reply:
Updated.